.travis/Dockerfile.fedora:rawhide

```diff
@@ -0,0 +1,23 @@
+FROM fedora:rawhide
+
+WORKDIR /build
+
+# Hack for DNF bug
+# ValueError("The supplied metadata version isn't supported")
+RUN sed -i s/enabled=1/enabled=0/ /etc/yum.repos.d/*-modular.repo || :
```

Comment:
Isn't this issue only with fedora:28 docker image? Looks like [it's fixed](https://bugzilla.redhat.com/show_bug.cgi?id=1566593) but the docker image is not updated yet.

BTW, any reason to skip testing on 28?